apis/cache/v1alpha2/zz_generated_terraformed.go

```diff
@@ -0,0 +1,396 @@
+/*
+Copyright 2020 The Crossplane Authors.
```

Comment:
Should we update the date in [boilerplate](https://github.com/crossplane-contrib/provider-jet-azure/blob/main/hack/boilerplate.go.txt)?